src/Mvc/Mvc.Testing/src/Microsoft.AspNetCore.Mvc.Testing.csproj

```diff
@@ -7,7 +7,7 @@
     <PackageTags>aspnetcore;aspnetcoremvc;aspnetcoremvctesting</PackageTags>
     <IsPackable>true</IsPackable>
     <!-- We're disable NU5100 explicitly bundling assemblies as tasks so they are not referenced when consumed. -->
-    <NoWarn>NU5100</NoWarn>
+    <NoWarn>$(NoWarn);NU5100</NoWarn>
```

Comment:
I realize this isn't caused by you, but I think we might be able avoid this warning by packing the binaries in the `build` directory rather than in a `task` folder. Would you be interested in taking a stab at that?

Reply:
Sure.

Do you have a rough link to some prior art elsewhere in the repo to get a gist of the changes needed?

Happy for me to do that separately rather than roll it into this?